src/Build.UnitTests/ProjectCache/ProjectCacheTests.cs

```diff
@@ -1656,5 +1659,73 @@ private void SetEnvironmentForErrorLocations(ErrorLocations errorLocations, Erro
                 }
             }
         }
+
+        [DotNetOnlyFact("The netfx bootstrap layout created with 'dotnet build' is incomplete")]
```

Comment:
Is this still true?
What happens with the test when run for core?

Reply:
Yes, the case failed with error 
_C:\Users\VssAdministrator\AppData\Local\Temp\ic12eptb.spp\MSBuildTemp\Temporaryd3b90dac326b4ce7bfb136f3617deafb\app.csproj : error MSB4246: SDK Resolver folder exists but without an SDK Resolver DLL or manifest file. This may indicate a corrupt or invalid installation of MSBuild. SDK resolver path: D:\a\1\s\artifacts\bin\bootstrap\net472\MSBuild\Current\Bin\SdkResolvers\Microsoft.DotNet.MSBuildSdkResolver\r\nDone Building Project_